docs/UserGuide.md

```diff
@@ -328,15 +328,15 @@ Is satisfied if `field`'s value is equal to `value`
 ### `inSet` _(field, values)_
 
 ```javascript
-{ "field": "type", "is": "inSet", "values": [ "X_092", 123, null, "2001-02-03T04:05:06.007" ] }
+{ "field": "type", "inSet": [ "X_092", 123, null, "2001-02-03T04:05:06.007" ] }
```

Comment:
Not a problem introduced here, but do we support an `inSet` key covering multiple different types (String, Integer and Date) as shown here?

Reply:
123 and null have been removed in latest commit